src/components/Projects/Projects.jsx

```diff
@@ -159,8 +159,8 @@ export class Projects extends Component {
             onClick={this.toggleProjectInfoModal}
           />
           <Overview numberOfProjects={numberOfProjects} numberOfActive={numberOfActive} />
-          {hasPermission(role, 'addProject', roles, userPermissions) ? (
-            <AddProject addNewProject={this.addProject} />
+          {hasPermission(role, 'postProject', roles, userPermissions) ? (
+            <postProject addNewProject={this.postProject} />
```

Comment:
The name of a component must begin with an uppercase letter `<PostProject ...> `.

Reply:
I agree

Reply:
I changed 'post' back to 'Add' for all of these. Let me know if I missed any.